.cirrus.yml

```diff
@@ -808,7 +811,10 @@ release_task:
 release_test_task:
     name: "Optional Release Test"
     alias: release_test
-    only_if: $CIRRUS_PR != ''
+    # Release-PRs are never ever marked with [CI:DOCS] or [CI:BUILD]
+    only_if: *not_build
```

Comment:
This is the test that sometimes people click on and it fails with "should not contain -dev", right?

Would it make sense to make this `only_if: $CIRRUS_TITLE contains the string "release"` ?

Anyhow, if that's possible, or if you need to repush for any reason, could you fix the typo in `preparation` below?

Otherwise, I can't really LGTM because this is way way way above my head, but if it passes CI I guess I'll approve it and we'll see what happens.

Reply:
> Would it make sense

Oh, that's an interesting idea, yep certainly possible.

> could you fix the typo

Certainly.

> this is way way way above my head

Mine too, really it's just "guess and see if it works" at this point.

Reply:
Update: In reality (and according to `RELEASE_PROCESS.md`) it appears like we need this on either `release` or `Bump` keywords (mostly the later).

Reply:
https://github.com/containers/podman/pull/13114